samples/nrf_rpc/protocols_serialization/client/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _nrf_rpc_protocols_serialization_client:
+
+Protocols serialization: client
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Protocols serialization client sample demonstrates how to send remote procedure calls (RPC) to a protocols serialization server device, such as one running the :ref:`Protocols serialization server <nrf_rpc_protocols_serialization_server>` sample.
+The RPCs are used to control Bluetooth® LE and :ref:`OpenThread <ug_thread_intro>` stacks running on the server device.
+The client and server devices use the :ref:`nrfxlib:nrf_rpc` and the :ref:`nrf_rpc_uart` to communicate with each other.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+To test the sample, you also need another device running the :ref:`Protocols serialization server <nrf_rpc_protocols_serialization_server>` sample.
+
+For testing the Bluetooth LE API serialization, you also need to have the `nRF Connect for Mobile`_ app installed on your smartphone or tablet.
+
+Overview
+********
+
+The Protocols serialization client sample is a thin client that neither includes OpenThread nor Bluetooth LE stacks. Instead, it provides
+implementations of selected OpenThread and Bluetooth LE functions that forward the function calls over UART to the Protocols serialization server.
+The client also includes a shell for testing the serialization.
+
+Configuration
+*************
+
+Snippets
+========
+
+* set the :makevar:`SNIPPET` option to ``ble;coex;debug;log_rpc;openthread``
+
+.. include:: /includes/sample_snippets.txt
```

Comment:
```suggestion
.. |snippet| replace:: :makevar:`client_SNIPPET`

.. include:: /includes/sample_snippets.txt
```
@greg-fer You're more familiar with snippets, but I think this is the correct way to define the variable for the include.

Reply:
First time I see this include file, so I'm totally unfamiliar with it. You seem way more familiar, because you reviewed the PR that introduced it: https://github.com/nrfconnect/sdk-nrf/pull/12909 :)

The formatting of the variable is correct, but the include has phrasing that points to Zephyr, which should be improved. But we can do this in a separate PR. https://nordicsemi.atlassian.net/browse/NCSDK-28722

Reply:
Yeah, the variable name was what I wasn't 100% on.